packages/utils/src/index.ts

```diff
@@ -4,8 +4,10 @@ import * as conversions from "./conversions"
 import * as packing from "./proof-packing"
 import * as typeChecks from "./type-checks"
 import * as errorHandlers from "./error-handlers"
+import * as cryptoNode from "./crypto/crypto.node"
+import * as cryptoBrowser from "./crypto/crypto.browser"
```

Comment:
```suggestion
import * as crypto from "./crypto/crypto.node"
```

There should be 1 import only here. The Rollup alias plugin replaces "./crypto/crypto.node" with "./crypto/crypto.browser" during the building process.

For example: https://github.com/semaphore-protocol/semaphore/blob/43f555e229fd25b02b3cacf0d1a9ca0b39b68c4a/packages/identity/src/identity.ts#L5